build.sbt

```diff
@@ -67,4 +71,11 @@ lazy val idGenerator = (project in file("idGenerator"))
   .settings(
     version := "1.0.0",
     name := "id-generator",
-    libraryDependencies += scalaLogging)
\ No newline at end of file
+    libraryDependencies ++= scalaLogging)
```

Comment:
That's what I intended (no idea why I mentioned `scala_2.11`, since there won't be a name clash or anything here). So, sounds good?